### PR TITLE
Simplify main file and data json

### PR DESCRIPTION
**Pull request by tridelat**
- [x] Moved most functions to read_json for creating seahowl system
- [x] AeroDyn options (filepaths) now in json files
- [x] Update README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,6 +129,7 @@ For discretization of blades and tower, it is possible to either use an ordered
 It is a JSON dictionary containing:
 - blades: (dict)
   - **fpm**: (bool) whether to consider Fully-Populated Matrix (FPM) elements (6x6 material properties) or not.
+  - **file_aerodyn**: file path of aerodyn file (relative to this file path), only used if aerodyn option is true in turbine file.
```
Comment (wenchaoyu) [inline README.md]: remove aerodyn from turbine.json in readme

File: tests/unit_tests/test_components.cpp [hunk 2]
```diff
@@ -30,7 +30,7 @@ using std::filesystem::path;
 using std::filesystem::absolute;
 
 static path DATADIR{};
-// static path DATADIR = path("../../data");
+// static path DATADIR = path("../../data/IEA15MW");
```
Comment (wenchaoyu) [inline tests/unit_tests/test_components.cpp]: remove the commented line

File: tests/unit_tests/test_components.cpp [hunk 3]
```diff
@@ -391,16 +391,16 @@ TEST(test_aerodyn, rpm_initial_pitch) {
...
     auto turbine = get_turbine_from_json(turbine_file);
     // remove controller
     turbine.controller = std::make_shared<seahowl::servo::Controller>();
 
     turbine.use_aerodyn = true;
 
-    turbine.aerodyn =
-            std::make_shared<seahowl::aero::AeroDynAdapter>((DATADIR / "aerodyn/IEA15MW/IEA-15-240-RWT_AeroDyn15.dat").generic_string(),
-                                                     (DATADIR / "aerodyn/IEA15MW/IEA-15-240-RWT_InflowWind.dat").generic_string());
+    turbine.aerodyn = std::make_shared<seahowl::aero::AeroDynAdapter>(
```
Comment (wenchaoyu) [inline tests/unit_tests/test_components.cpp:400]: remove the hardcoded path for IEA15MW

File: data/IEA15MW/main.json [hunk 4]
```diff
@@ -27,7 +27,10 @@
     {
       "translation": [0, 0, 0],
       "rotation": 0,
-      "file": "./IEA15MW_turbine.json"
+      "file": "./turbine.json",
+      "use_aerodyn": true,
```
Comment (wenchaoyu) [inline data/IEA15MW/main.json]: put use_aerodyn as false by default

**Review by wenchaoyu — CHANGES_REQUESTED** [review]
I run several tests with or without aerodyn, things work well.

File: tests/unit_tests/test_components.cpp (hunk 3 above)
Comment (tridelat) [reply tests/unit_tests/test_components.cpp:400]: Actually this is a test, so we need to give it an input file (turbine) that does not have the aerodyn info (from main file that is not read here)

**Review by wenchaoyu — APPROVED** [review]

**Pull request merged**